### DISCUSSION TITLE
Issues starting dump1090 up (wont connect to the dongle)

### DISCUSSION THREAD

**Discussion by CraigglesO** [rsadsb/dump1090_rs#64] [General]
After hours of tinkering, I finally stumbled upon the answer why I couldn't connect after following the tutorial:

https://www.reddit.com/r/RTLSDR/wiki/blacklist_dvb_usb_rtl28xxu/

Sharing for others as this fixes against modern linux issues.

**Comment by gariac** [comment]
Most of us here have already done the blacklisting because we already run the "normal" version of dump1090.  However I could see a windows user switching to linux or a person who has never used a rtlsdr on linux would need to know about the blacklist.  Yeah this would be really hard to debug, but now you are a better person!
